src/pages/managerForm.js

```diff
@@ -1,66 +1,387 @@
+                    let listing = []
+                    snapshot.docs.forEach((doc) => {
+                        listing.push({ ...doc.data(), id: doc.id})
+                    })
+                    console.log(listing)
+                })
+                .catch(error => {
+                    console.error(error.message)
+                });
+
+            const docRef = async () => {
+                await addDoc(commCollectionRef, {
+                    community_name: setFormData.community_name.value,
+                    community_manager: setFormData.community_manager.value,
+                    community_date: setFormData.community_date.value,
+                    community_location: setFormData.community_location.value,
+                    community_url: setFormData.community_url.value,
+                    community_linkedin: setFormData.community_linkedin.value,
+                    community_twitter: setFormData.community_twitter.value,
+                    community_facebook: setFormData.community_facebook.value,
+                    community_vision: setFormData.community_vision.value,
+                    community_description: setFormData.community_description.value,
+                })
+                .then(() => {
+                    setFormData.reset("")
+                })
+                    console.log("Document writtern with ID:", docRef.id)
+            }
+
+        } 
+
+    const imageUpload = async (file) => {
+        // if (!file) return;
+
+        try {
+            const imageRef = firebase.storage().ref();
+            const fileName = `${Date.now()}_${file.name}`;
+            const fileRef = imageRef.child(fileName);
+            await fileRef.put(file);
+            const downloadURL = await fileRef.getDownloadURL();
+            return downloadURL;
+        } catch (error) {
+            console.error("Error uploading image:", error)
+            throw error;
+        }
+    } 
+
+    const uploadFile = async (event, file) => {
+        event.preventDefault()
+        try {
+            const downloadURL = await imageUpload(file)
+            console.log("Image uploaded! Download URL:", downloadURL)
+        } catch (error) {
+            console.error("Error uploading image", error)
+        }
+    };
+
+    const deletFile = () => {
+        if (fileInputRef.current) {
+            fileInputRef.current.value = '';
+        }
+    }
+
+    const goHome = (event) => {
+        event.preventDefault();
+        <Link href="/" />
+    }
+
+    const fileInputRef = useRef(null);
+    const deleteFile = useRef(null) 
+    
+
     return (
         <div>
             <NavBar />
-            <h1 className={styles.textColor}>managerForm!!!</h1>
-            <div className={styles.firstBanner}>New Community</div>
+            <h1 className={styles.firstBanner}>New Community</h1>
             <div className={styles.secondBanner}>
                 <h2>Choose category(s)</h2>
```

Comment:
Do we have the field in the database for this "Choose Category" question? If not, please omit for the MVP. 🙏🏾 
If we do have this field in the DB, are we going to allow for managers to create their own categories if none of t suggested categories fit? If not, then we should have an "Other" option.

I'm fine with either option; I just have the end-of-June in sight 😅

Reply:
I see your point. Removed.